Clean Swift/Unit Tests (Quick-Nimble).xctemplate/___FILEBASENAME___InteractorSpec.swift

```diff
@@ -109,25 +118,25 @@ class ___VARIABLE_sceneName___InteractorSpec: QuickSpec {
                 sut.perform___VARIABLE_sceneName___(with: request)
 
                 // then
-                expect(presentationLogicSpy.presentPerform___VARIABLE_sceneName___Called).to(beTrue())                
-            })            
+                expect(presentationLogicSpy.presentPerform___VARIABLE_sceneName___Called).toEventually(beTrue())
```

Comment:
Any reason why it needs to _always_ be `eventually`?

Reply:
@zaimramlan because on actual implementation, 90% of the time it will be an actual API call...